Watchman.DomainModel/Settings/GetConfigurationQueryHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Watchman.Cqrs;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings
+{
+    public class GetConfigurationQueryHandler : IQueryHandler<GetConfigurationQuery, GetConfigurationQueryResult>
+    {
+        private readonly ISessionFactory _sessionFactory;
+
+        public GetConfigurationQueryHandler(ISessionFactory sessionFactory)
+        {
+            this._sessionFactory = sessionFactory;
+        }
+
+        public GetConfigurationQueryResult Handle(GetConfigurationQuery query)
```

Comment:
dobrze będzie tutaj przekazywać server ID -> dla którego serwera chcemy pobrać tą konfiguracje

Reply:
u mnie każde server ID zapisane w bazie ma wartość 0, czy tak powinno być? bo jeśli tak, to po co je przekazywać? 

![image](https://user-images.githubusercontent.com/35144936/93810033-5dd35100-fc4e-11ea-9c7d-3305577d5952.png)

Reply:
Ponieważ te są domyślne, ale jak dany serwer nadpisze konfiguracje dla siebie to już otrzyma odpowiedni ServerId

Reply:
ID 0 jest dla domyślnych konfiguracji 
serwer może nadpisać konfiguracje 

jeśli nadpisał - używa nadpisanej
jeśli nie nadpisał - używa domyślnej z ServerID = 0

Reply:
dlatego potrzebna jest logika

jeśli istnieje obiekt który pasuje do tego ServerID -> chcemy go pobrać
a jeśli nie, to bierzemy ten z ServerID 0

Reply:
a, w ten sposób :) dzięki

Reply:
ok, a czy może się zdarzyć, że w bazie będą dwa obiekty określające daną konfigurację, ale jeden będzie mieć id=0, a drugie ID jakiegoś serwera, który go używa? chciałabym wiedzieć, jak bardzo wyszukana ma być ta logika xd

Reply:
`w bazie będą dwa obiekty określające daną konfigurację, ale jeden będzie mieć id=0, a drugie ID jakiegoś serwera, który go używa`

właśnie tak to wygląda jak serwer ma swoją, niestandardową konfiguracje